packages/entity-server/src/__tests__/__integration__/EntityRelationshipsRoutes.test.ts

```diff
@@ -3,21 +3,19 @@
  * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
  */
 
-import { clearTestData, getTestDatabase } from '@tupaia/database';
 import { TestableServer } from '@tupaia/server-boilerplate';
-import { grantAccessToCountries, revokeCountryAccess, setup } from '../testUtilities';
+import { grantAccessToCountries, revokeCountryAccess, setupTestApp } from '../testUtilities';
 import { COUNTRIES } from './fixtures';
 
 describe('relationships', () => {
   let app: TestableServer;
 
   beforeAll(async () => {
-    app = await setup();
+    app = await setupTestApp();
     grantAccessToCountries(COUNTRIES);
```

Comment:
`grantAccessToCountries()` is more like `setupAccessPolicyForUserMock()` or `setupMock()`.

Reply:
Hmmm I see your point, but to me that seems more like an implementation detail? I like how with the current name it just tells you what it does, not how it does it 👍

Reply:
I'm happy with the current name to keep using the original one. 
I guess what I was trying to say is at my first glance I feel like `grantAccessToCountries()` should be part of `setupTestData()`, but apparently it has a different meaning. 😄